ui_components/src/village/Village.js

```diff
@@ -16,34 +16,34 @@ function Village({
     const [resourceDataState, setResourceDataState] = React.useState(resourceData);
     const [playerSeatState, setPlayerSeatState] = React.useState(playerSeat);
     const [modalState, setModalState] = React.useState("closed");
-    const resourceDays = React.useRef(1);
-    const resourceDaysDisplay = React.useRef("daily");
+    const [resourceDays, setResourceDays] = React.useState(1);
```

Comment:
One more ask, reading this again I'm not sure what "resourceDays" is supposed to represent, could we use a more specific name like `resourceDaysToShow`